com.unity.render-pipelines.universal/Documentation~/universalrp-asset.md

```diff
@@ -95,6 +95,8 @@ This section allows you to fine-tune global post-processing settings.
 
 | Property         | Description                                                  |
 | ---------------- | ------------------------------------------------------------ |
+| __Post Processing__ | Check this box to enable post processing in your scene. This includes stripping post processing shaders and textures. <br/>**Note:** It will not strip assets if they are referenced somewhere else in scene or even in other URP Asset. |
```

Comment:
"in your scene" - remove.
"This includes stripping post processing shaders and textures." - "If you clear this check box, Unity excludes post-processing shaders and textures from the build."
"**Note:** It will not strip assets if they are referenced somewhere else in scene or even in other URP Asset." - "**Note:** Unity does not exclude assets related to post-processing in one of the following cases:
* Other assets in the build refer to the assets related to post-processing.
* A different URP asset has the Post-processing property enabled."